src/locales/en/en.json

```diff
@@ -869,7 +870,18 @@
       "header": "New connection",
       "cancel": "Cancel",
       "scanqr": "Scan QR code",
```

Comment:
I would refactor the structure of the "menu" to have nested levels as the picture below.

Something like this:

```
{
  "menu": {
    "header": "Menu",      
      "items": {
        "profile": {
          "title": "Profile"
        },        
        "crypto": {
          "title": "Crypto"
        },
        "connections": {
          "title": "Connections"
        },
        "connectwallet": {
          "title": "Connect Wallet",
          "subtitle": "CIP-45",
          "section": {
            "header": "Cardano Connect",
            ...
          }
        }
      }
  }
}
```

![10052024103406](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/53ce9e97-2390-48e0-95f8-17176bee32e2)